### PR TITLE
feat: website layout

### PR DESCRIPTION
**Pull request by vermasavita**
1. Built layout page for UI Book
2. Created two button for two different action

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.head-content{
+    margin: auto auto auto auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 60%;
+    margin: auto;
```
Comment (VishalPatil18) [inline css/main.css]: Margin is used twice here. You could drop one of these. I would suggest dropping the first one.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    margin-top: 12rem;
+   
+
```
Comment (VishalPatil18) [inline css/main.css]: You could avoid empty lines here.

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    font-family:cursive;
+}
+.head-content h1{
+    font-size: 6rem;
+    margin-bottom:1.8rem;
+    text-align: center;
+    width: 95%;
+    font-weight: 900;
+    color: var(--text-color);
+    font-family:fantasy
```
Comment (VishalPatil18) [inline css/main.css]: A `;` missing after `font-family:fantasy` You could fix this.

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    background-color: var(--secondary-color);
+    box-shadow: 0 8px 12px var(--bg-secondary-color);
+    border: 1px solid var(--primary-color);
+    border-radius: 5px;
+    color: var(--primary-color);
+    font-size: 2rem;
+    padding: .5rem;
+    text-decoration: none;
+    margin-right: 2rem;
+    padding: 1rem;
```
Comment (VishalPatil18) [inline css/main.css]: Padding is used twice which makes the second appearance overwrite the previous one. You could drop one of these.

**Comment by shaik-saad** [comment]
Please add a description as Atul has described.

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
- A couple of suggestions were added regarding code repetition. 
- A missing semicolon fix.
 
The UI looks quite clean and good now. Great work @vermasavita 👍

**Comment by vermasavita** [comment]
@shaik-saad Added. Is that good way to write?

File: css/main.css (hunk 1 above)
Comment (vermasavita) [reply css/main.css]: Noted. Made the changes.

File: css/main.css (hunk 2 above)
Comment (vermasavita) [reply css/main.css]: Removed

File: css/main.css (hunk 3 above)
Comment (vermasavita) [reply css/main.css]: Fixed this

File: css/main.css (hunk 4 above)
Comment (vermasavita) [reply css/main.css]: Removed

**Comment by shaik-saad** [comment]
> @shaik-saad Added. Is that good way to write?

You can add gify or a sreenshot of the features you have created, that'll help reviewers to know which one's are created in this pr.

**Comment by shaik-saad** [comment]
Also have you made the changes @VishalPatil18 has recommended?

**Comment by vermasavita** [comment]
For that purpose, I have added Netflify link. 

Yes, made the changes that Vishal has suggested.

**Comment by shaik-saad** [comment]
> For that purpose, I have added Netflify link.
> 
> Yes, made the changes that Vishal has suggested.

Oh, okay. Then you are good to merge.

**Review by shaik-saad — APPROVED** [review]

**Review by VishalPatil18 — APPROVED** [review]

**Pull request merged**